include/internal/libspdm_common_lib.h

```diff
@@ -595,6 +595,42 @@ bool libspdm_generate_cert_chain_hash(libspdm_context_t *spdm_context,
 bool libspdm_generate_public_key_hash(libspdm_context_t *spdm_context,
                                       uint8_t *hash);
 
+/**
+ * This function verifies the integrity of peer certificate chain buffer including
+ * spdm_cert_chain_t header.
+ *
+ * @param  spdm_context                  A pointer to the SPDM context.
+ * @param  cert_chain_buffer              Certificate chain buffer including spdm_cert_chain_t header.
+ * @param  cert_chain_buffer_size          size in bytes of the certificate chain buffer.
+ * @param  is_requester                   Indicates if it is a requester message.
+ *
+ * @retval true  Peer certificate chain buffer integrity verification passed.
+ * @retval false Peer certificate chain buffer integrity verification failed.
+ **/
+bool libspdm_verify_peer_cert_chain_buffer_integrity(libspdm_context_t *spdm_context,
+                                                     const void *cert_chain_buffer,
+                                                     size_t cert_chain_buffer_size,
+                                                     bool is_requester);
```

Comment:
`is_requester` is not needed. Its value is stored in the `spdm_context`.

Reply:
I forgot we have added the `is_requester` in `spdm_context`. I have changed the code. Thanks.